Tests/SourceKitLSPTests/CompilationDatabaseTests.swift

```diff
@@ -43,7 +43,9 @@ final class CompilationDatabaseTests: XCTestCase {
     let compilationDatabaseUrl = ws.builder.buildRoot.appendingPathComponent("compile_commands.json")
 
     _ = try ws.sources.edit({ builder in
-      let compilationDatabase = try JSONCompilationDatabase(file: AbsolutePath(validating: compilationDatabaseUrl.path))
+      let compilationDatabase = try JSONCompilationDatabase(
+        file: AbsolutePath(validating: compilationDatabaseUrl.path)
+      )!
       let newCommands = compilationDatabase.allCommands.map {
```

Comment:
Could you use `XCTUnwrap` instead of `!` here so the test doesn’t crash if the database is `nil`?